### PR TITLE
Use workspace limit idle timeout value in WorkspaceActivityManager

### PR DESCRIPTION
**Pull request by mkuznyetsov**
### What does this PR do?
WorkspaceActivityManager is now using accounts resources to retrieve workspaces idle timeouts. "che.limits.workspace.idle.timeout" property is also used now as default value for timeout.
Removed old "che.workspace.agent.dev.inactive_stop_timeout_ms" property.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/8951

### CODE REVIEW

**Review by mshaposhnik — APPROVED** [review]

**Comment by mshaposhnik** [comment]
Pls write more meaningful title.

**Review by benoitf — APPROVED** [review]

**Comment by mkuznyetsov** [comment]
ci-test

File: assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che_aliases.properties [hunk 1]
```diff
@@ -32,3 +32,5 @@ che.infra.kubernetes.installer_server_max_port=che.infra.openshift.installer_ser
 
 che.infra.docker.daemon_url=docker.client.daemon_url
 che.infra.docker.certificates_folder=docker.client.certificates_folder
+
+che.limits.workspace.idle.timeout=che.workspace.agent.dev.inactive_stop_timeout_ms
```
Comment (skabashnyuk) [inline assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che_aliases.properties]: new line

File: assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che_aliases.properties [hunk 2]
```diff
@@ -32,3 +32,5 @@ che.infra.kubernetes.installer_server_max_port=che.infra.openshift.installer_ser
 
 che.infra.docker.daemon_url=docker.client.daemon_url
 che.infra.docker.certificates_folder=docker.client.certificates_folder
+
```
Comment (skabashnyuk) [inline assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che_aliases.properties:36]: comment why we need it.

**Review by sleshchenko — APPROVED** [review]

**Review by skabashnyuk — APPROVED** [review]

**Review by riuvshin — APPROVED** [review]
does alias will guarantee that users who still using `CHE_WORKSPACE_AGENT_DEV_INACTIVE__STOP__TIMEOUT__MS` will have proper timeout ?

**Review by garagatyi — APPROVED** [review]

**Comment by mkuznyetsov** [comment]
@riuvshin no, it doesn't.

**Pull request merged**